docs/topics_5/proc_web-obtaining-latest-image.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+
+[id="proc_web-obtaining-latest-image_{context}"]
+= Obtaining the latest image
+
+The first step in troubleshooting in an OpenShift Container Platform environment is to be sure you are using the latest image.
+
+NOTE: If you have deployed the {WebName} by pasting in a JSON template, no image stream is created and the following steps are not applicable.
```

Comment:
@RichardHoch  "pasting in a JSON template" is too colloquial. Change to "importing a template". We need to be consistent. The templates are not called JSON templates in other parts of the doc, I think.